docs/release-notes/cloud/cloud-2022-08.md

```diff
@@ -0,0 +1,101 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud August 2022.
+included_jira_versions: ['2022.Q3.4', '2022.Q3.5', '2022.Q3.6']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/6.2.24
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/6.2.59
+---
+
+# Cloud August 2022
+
+These release notes are for the Codacy Cloud updates during August 2022.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   The [GitHub integration](../../repositories-configure/integrations/github-integration.md) now uses a GitHub Apps token instead of a personal one so that the following features continue working even if the person who created the integration leaves the Codacy organization:
+
+    -   Analyzing new commits and pull requests
+    -   Updating the status check of pull requests
+    -   Suggesting changes
+    -   Creating comments and issues on GitHub for issues detected by Codacy
+
+    Additionally, all activity related to the GitHub integration features now appear as having been performed by Codacy instead of the person who created the integration. (PLUTO-36, PLUTO-77)
+
+    ![Suggested fix created by the Codacy GitHub integration](../images/pluto-77.png)
```

Comment:
@lventura-codacy I adapted this release note from your announcement on Slack. Could you please check if you agree with this version that covers both [PLUTO-36](https://codacy.atlassian.net/browse/PLUTO-36) and [PLUTO-77](https://codacy.atlassian.net/browse/PLUTO-77)?

Reply:
Looks great to me @prcr ! I just added a small change suggestion, which you can feel free to ignore.